docs/data/apis/api-providers.mdx

```diff
@@ -13,24 +13,27 @@ Multiple infrastructure providers have made Stellar RPC and Horizon services ava
 
 These providers allow access to the Futurenet, Testnet and Mainnet network.
 
-| Provider | Futurenet | Testnet | Mainnet | Horizon | RPC |
-| --- | --- | --- | --- | --- | --- |
-| [Blockdaemon\*†](https://www.blockdaemon.com/apply/soroban) | ❌ | ✅ | ✅ | ✅ | ✅ |
-| [Validation Cloud\*†](https://app.validationcloud.io/) | ❌ | ✅ | ✅ | ✅ | ✅ |
-| [QuickNode\*](https://www.quicknode.com/docs/stellar) | ❌ | ✅ | ✅ | ✅ | ✅ |
-| [NowNodes](https://nownodes.io/nodes/stellar-xlm) | ✅ | ✅ | ✅ | ❌ | ✅ |
-| [Gateway](https://gateway.fm/public-rpc/) | ❌ | ✅ | ✅ | ❌ | ✅ |
-| [Ankr](https://www.ankr.com/rpc/advanced-api/) | ❌ | ✅ | ✅ | ✅ | ✅ |
-| [Infstones](https://infstones.com/) | ❌ | ❌ | ✅ | ❌ | ✅ |
-| [Obsrvr](https://www.withObsrvr.com/) | ❌ | ✅ | ✅ | ✅ | ✅ |
-| [GetBlock](https://getblock.io/nodes/xlm/) | ❌ | ❌ | ✅ | ❌ | ✅ |
-| [Nodies](https://nodies.org) | ❌ | ✅ | ✅ | ✅ | ✅ |
-| [OnFinality](https://onfinality.io/networks/stellar) | ❌ | ❌ | ✅ | ❌ | ✅ |
+| Provider | Futurenet | Testnet | Mainnet | Horizon | RPC | RPC Historical Access ** |
+| --- | --- | --- | --- | --- | --- | --- |
+| [Blockdaemon\*†](https://www.blockdaemon.com/apply/soroban) | ❌ | ✅ | ✅ | ✅ | ✅ | ❌ |
+| [Validation Cloud\*†](https://app.validationcloud.io/) | ❌ | ✅ | ✅ | ✅ | ✅ | ❌ |
+| [QuickNode\*](https://www.quicknode.com/docs/stellar) | ❌ | ✅ | ✅ | ✅ | ✅ | ❌ |
+| [NowNodes](https://nownodes.io/nodes/stellar-xlm) | ✅ | ✅ | ✅ | ❌ | ✅ | ❌ |
+| [Gateway](https://gateway.fm/public-rpc/) | ❌ | ✅ | ✅ | ❌ | ✅ | ❌ |
+| [Ankr](https://www.ankr.com/rpc/advanced-api/) | ❌ | ✅ | ✅ | ✅ | ✅ | ❌ |
+| [Infstones](https://infstones.com/) | ❌ | ❌ | ✅ | ❌ | ✅ | ❌ |
+| [Obsrvr](https://www.withObsrvr.com/) | ❌ | ✅ | ✅ | ✅ | ✅ | ✅ |
+| [GetBlock](https://getblock.io/nodes/xlm/) | ❌ | ❌ | ✅ | ❌ | ✅ | ❌ |
+| [Nodies](https://nodies.org) | ❌ | ✅ | ✅ | ✅ | ✅ | ❌ |
+| [OnFinality](https://onfinality.io/networks/stellar) | ❌ | ❌ | ✅ | ❌ | ✅ | ❌ |
+| [Quasar](https://quasar.lightsail.network/) | ❌ | ❌ | ❌ | ❌ | ✅ | ✅ |
 
 \*_Blockdaemon, Validation Cloud, and Quicknode combine Horizon and RPC in the same endpoint._
 
 †_Blockdaemon and Validation Cloud provide full historical data for Horizon._
 
+**_RPC Historical Access is a new option for those looking to retrieve full ledger history. Currently only the [getLedgers](https://developers.stellar.org/docs/data/apis/rpc/api-reference/methods/getLedgers) RPC method supports this feature. You can choose one of the providers above, or create your own getLedgers archive by following [these steps](https://developers.stellar.org/docs/data/apis/rpc/admin-guide/data-lake-integration)._
```

Comment:
@johncanneto This is not in italics. You may have to escape the **
See line 31:
\**_RPC........